<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>电影资源搜索器        By Ramond.Z</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLineEdit" name="keywordText"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="searchButton">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QCheckBox" name="heaven">
      <property name="text">
       <string>来源1：电影天堂</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QCheckBox" name="hd_radio">
      <property name="text">
       <string>来源2：高清电台(资源贼多，搜索贼慢)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="btZhijia">
      <property name="text">
       <string>来源3：BT之家</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="hd_mp4">
      <property name="text">
       <string>来源4：高清MP4吧</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QCheckBox" name="chinese_hd">
      <property name="text">
       <string>来源5：中国高清网</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QTableWidget" name="resultTable"/>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="searchResult">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>searchButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>653</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resultTable</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>cell_clicked(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>searchButton_clicked()</slot>
  <slot>cell_clicked(int,int)</slot>
 </slots>
</ui>
